Journeys/Email integration/Promise date change communication.md

```diff
@@ -0,0 +1,85 @@
+# Integration guidelines for OMS and Marketing Automation Platforms: Promise Date Change Communication 
+
+
+## Overview
+This document outlines the steps to ensure accurate promise date change communication from the Order Management System (OMS) to the Marketing Automation Platform. This integration enhances the customer experience and reduces manual efforts.
+
+### Step 1: Identify Promise Date Changes
+
+The bulkSendPromiseDateChange service in OMS identifies all the sales order items whose promise dates have changed in throughout the day. The frequency of this service is configurable as per requirement, but it is recommended to set it once a day.
```

Comment:
Check grammar

Reply:
Made the changes. Removed 'in' before throughout